### PR TITLE
Fix/add alt for images

### PR DESCRIPTION
**Pull request by JacOng17**
Added alt for images.

Added `aria-label` for alt text.

### CODE REVIEW

File: _includes/card.html [hunk 1]
```diff
@@ -15,7 +15,8 @@
          class="card-image-bg {% unless include.image_src %} card-image-svg {% endunless %}"
          {% if include.image_src %}
          style="background-image: url({{ site.baseurl }}{{ include.image_src }});"{% endif %}>
-      {% unless include.image_src %}
+        <p tabindex="8" class="aria-hidden">{{ include.image_alt }}</p>
```
[the remark this answers is not in the document]
Comment (dlouise64) [reply _includes/card.html]: @JacOng17 `tabindex="0" should work here`

File: _services_projects/legislation-as-code.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ permalink: /projects/legislation-as-code/
 redirect_from: /projects/legislation-as-code/
 excerpt: Exploring the benefits of legislation that is both human and machine readable and what it might take to transition.
 image: /assets/img/projects/legislation-as-code.png
-image_accessibility: 
+image_accessibility: Image of OpenFisca & how it calculates an output using rules & formulas.
```
Comment (Br3nda) [inline _services_projects/legislation-as-code.md]: Can we describe how Open Fisca calculates an output using rules and formulas? To ensure the blind user gets the same information a sighted user would.

File: _services_projects/notice-of-sale.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ permalink: /projects/notice-of-sale/
 redirect_from: /projects/notice-of-sale/
 excerpt: Understand the Notice of Sale process as part of the ‘selling and buying a property’ life event and potential improvements for people and agencies involved.
 image: /assets/img/projects/notice-of-sale.png
-image_accessibility:
+image_accessibility: Image of a house and a handshake signifying an agreement.
```
Comment (Br3nda) [inline _services_projects/notice-of-sale.md]: :sparkles:

File: _services_projects/optimistic-futures.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ permalink: /projects/optimistic-futures/
 redirect_from: /projects/optimistic-futures/
 excerpt: Speculating how government, public institutions and society could be resilient and prosperous in the future.
 image: /assets/img/projects/optimistic-futures.png
-image_accessibility: 
+image_accessibility: Image of Lego figures.
```
Comment (Br3nda) [inline _services_projects/optimistic-futures.md]: Can you add what the Lego figurines are doing? So the blind user gets all the same info a sighted user does.

**Pull request merged**